src/platform/user/authentication/config/constants.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable camelcase */
-export const VAMockProvider = {
+/* eslint-disable-next-line no-unused-vars */
```

Comment:
@ianmagenta I didn't realize we weren't using VAMockProvider anywhere.. we can remove that dead code.